holopy/scattering/theory/tmatrix_f/makefile

```diff
@@ -0,0 +1,23 @@
+#makefile to make S.exe, which is called by HoloPy to run Tmatrix scattering calculations
+
+ifdef COMSPEC
+	#Windows
+	RM=del S.lp.o lpq.o
+else
+	#Not Windows
+	RM = rm S.lp.o lpq.o
+endif
+
+#make files
+S.exe : S.lp.o lpq.o
+	gfortran -o S.exe S.lp.o lpq.o
```

Comment:
leave off the ".exe" here.  `gfortran -o S S.lp.f lpq.f` will compile the executable and automatically give it the appropriate platform-specific extension (no extension on Linux, .exe extension on Windows).